File: Dummy/hawkshaw/library/BatchThumbUploadResponseOrBuilder.java
package com.hawkshaw.library;

import com.google.protobuf.C0369n0;
import com.google.protobuf.C0371o0;
import com.hawkshaw.library.BatchThumbUploadResponse;
import java.util.List;

public interface BatchThumbUploadResponseOrBuilder extends C0371o0 {
    /* synthetic */ C0369n0 getDefaultInstanceForType();

    BatchThumbUploadResponse.Thumb getSuccess(int i5);

    int getSuccessCount();

    List<BatchThumbUploadResponse.Thumb> getSuccessList();

    /* synthetic */ boolean isInitialized();
}

File: app/src/main/java/com/hawkshaw/library/BatchThumbUploadResponseOrBuilder.kt
package com.hawkshaw.library

/**
 * Interface for accessing BatchThumbUploadResponse properties.
 */
interface BatchThumbUploadResponseOrBuilder {
    /**
     * Get a specific thumb from the success list
     * @param index zero-based index of the element to retrieve
     * @return Thumb instance at the specified index
     */
    fun getSuccess(index: Int): BatchThumbUploadResponse.Thumb

    /**
     * Get the number of successful thumbnail uploads
     * @return count of successful thumbnails
     */
    fun getSuccessCount(): Int

    /**
     * Get the list of successful thumbnail uploads
     * @return unmodifiable list of Thumb instances
     */
    fun getSuccessList(): List<BatchThumbUploadResponse.Thumb>
} 